### ISSUE TITLE
Initializing a simulation with a VTU file does not work

### ISSUE THREAD

**Issue by ktbolt** [SimVascular/svFSI#108]
I've been trying to restart a simulation using the following commands
```
Continue previous simulation: t
Simulation initialization file path: result_500.vtu
Starting time step: 500
```
The `result_500.vtu` reads in fine but the simulation starts at time step 1 rather than time step 501.

I think the problem is that the `INITFROMVTU` function that reads in the VTU file does not set the `cTS` or `time` globals like the `INITFROMBIN` functions does.

**Comment by ktbolt** [comment]
I set the set the `cTS` and` time` globals in the `INITFROMVTU` function and the simulation now seems to initialize correctly.

**Comment by vvedula22** [comment]
@ktbolt did you set the `time` parameter to `0` or the equivalent of `cTS * dt`? BTW, we don't usually use this feature (loading a vtu file) for restarting a simulation. Instead, we use a bin file as it has all the necessary information, including residues. Initializing from vtu is intended for providing a better initial guess of the state variables when the `time` variable could still be set to `0`.

**Comment by ktbolt** [comment]
@vvedula22 I'm not sure what you mean by `time` parameter, is there such a parameter you can set in the .inp file? Or do you mean the global `time` variable in `MOD.f`?

The user wanted to restart a simulation using a different number of processors, did not appear you could do that with a bin file so I was looking are using a VTU file.

**Comment by vvedula22** [comment]
@ktbolt I meant the global `time` variable. 

Yes, restarting from a bin file will not allow you to change the number of processors. But if a user decides to change the time step size and chooses vtu initialization, the `time ` variable will be incorrectly computed. 

Should we also provide an option to provide a simulation start time as input parameter?

**Comment by ktbolt** [comment]
@vvedula22 Good point! We could add a `Starting time` parameter, similar to `Starting time step` one I suppose.

**Comment by vvedula22** [comment]
Solved in pull request #110
